routes/rooms.go

```diff
@@ -141,12 +141,32 @@ func UpdateEmergencyRoom(ctx *gin.Context) {
 // @Produce    json
 // @Param      authorization                   header      string                true    "Bearer: <TOKEN>"
 // @Param      id                              path        string                true    "Emergency Room's ID"
-// @Success    200                             {object}    StatusResponse
```

Comment:
AFAIK this is the only place where we used the StatusResponse struct, we should remove it.

Reply:
#55